src/app/CodeRepo/lib/types/revision.js

```diff
@@ -119,9 +119,18 @@ class Revision extends Type {
     }
 
     async skipReview() {
-        this.tags = this.tags.filter((tag) => !tag.startsWith("review:"));
+        await this.clearTags("review:", false)
         this.tags.push(`review:skip`);
-        await this.save()
+        await this.save();
+    }
+
+    async clearTags(pattern, save=true) {
```

Comment:
missing spaces around =
Should not this be a method on the Type-object so all types can benefit from it?